tests/performance/programmatic_feed_client/programmatic_feed_client.rb

```diff
@@ -65,14 +65,14 @@ def setup
         :x => 'loadgiver',
         :y => 'feeder.throughput',
         :filter => {'loadgiver' => VESPA_FEED_CLIENT, 'size' => LARGE },
-        :y_min => 34000, :y_max => 40000,
+        :y_min => 33000, :y_max => 41000,
         :historic => true
       },
       {
         :x => 'loadgiver',
         :y => 'feeder.throughput',
         :filter => {'loadgiver' => VESPA_JSON_FEEDER, 'size' => LARGE },
-        :y_min => 33000, :y_max => 38000,
+        :y_min => 39000, :y_max => 39000,
```

Comment:
min == max ?

Reply:
Yes, wee expect very consistent results for this ... "cough"